gcp_pilot/drive.py

```diff
@@ -1,16 +1,55 @@
 # Reference: https://developers.google.com/drive/api/reference/rest/v3
 
-from gcp_pilot.base import GoogleCloudPilotAPI
-
+from gcp_pilot.base import DiscoveryMixin, GoogleCloudPilotAPI
 
 class Drive(GoogleCloudPilotAPI):
```

Comment:
Since `Drive` cannot be used directly, maybe extend from abc.ABC?

Reply:
actually it can be used directly. Someone can initialize and use the `service.client`

Reply:
But without any methods, it's useless, right?

Reply:
well kind of, I would say. It might not be recommended to use directly given our coding-style and purpose.. but as I mentioned before it could still be useful for someone who wants to use a non supported Drive service

we have two options - just to make sure their W/S
a) extends from abc: by preventing "standalone" usage of Drive
or 
b) do not extends: allow anyone to use Drive freely

I don't have preferences, I’m slightly inclined toward option b

Reply:
I'd lean towards (a) to align with gcp-pilot's purpose: friendlier wrapper.
Anyone can still use the Drive if it extends to its own class, just like you are already doing with some resources.